dotcom-rendering/cypress/integration/parallel-3/liveblog.interactivity.spec.js

```diff
@@ -145,7 +145,7 @@ describe('Liveblogs', function () {
 			},
 			(req) => {
 				expect(req.query.lastUpdate).to.equal(
-					'62148e2d8f081f9e465d1bb7',
+					'block-62148e2d8f081f9e465d1bb7',
```

Comment:
Why did this need updating?

Reply:
A change on frontend. Previously frontend sent the id without the block prefix but now it sends it with the prefix so any branches with the old expectation need updating.